<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountViewer</class>
 <widget class="QWidget" name="AccountViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="pm_group">
     <property name="title">
      <string>Payment Methods</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="payment_methods"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pm_buttons">
        <item>
         <widget class="QPushButton" name="pm_add">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/credit-card-plus.svg</normaloff>:/resources/material-design/svg/credit-card-plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pm_edit">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pm_remove">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/credit-card-off.svg</normaloff>:/resources/material-design/svg/credit-card-off.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="scheduled_ops_group">
     <property name="title">
      <string>Scheduled Operations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="sops_table">
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sops_buttons">
        <item>
         <widget class="QPushButton" name="sops_add">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/calendar-plus.svg</normaloff>:/resources/material-design/svg/calendar-plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sops_edit">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sops_remove">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/calendar-remove.svg</normaloff>:/resources/material-design/svg/calendar-remove.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
